### DISCUSSION TITLE
New Stable Kernel 6.4.10-ophub -- Is it for all SOC or is it just for Allwiner?

### DISCUSSION THREAD

**Discussion by JFLim1** [ophub/amlogic-s9xxx-armbian#1647] [Amlogic]
Hi @ophub,

Noticed a new kernel in your stable-kernel download page https://github.com/ophub/kernel/releases/download/kernel_stable/6.4.10.tar.gz.  Is this kernel suitable for Amlogic S922X?

Reason for inquiring is only "dtb-allwinner-6.4.10-ophub" is included with this kernel.

**Comment by ophub** [comment]
just TQC-A01(h6)

**Comment by JFLim1** [comment]
Hi @ophub,

Thanks for the info.

Any plans to release kernel 6.4 or 6.5 for Amlogic SOC?

**Comment by ophub** [comment]
I will not maintain the kernel, these are shared by others.
